src/internal/packager/images/pull.go

```diff
@@ -331,12 +330,7 @@ func CleanupInProgressLayers(ctx context.Context, img v1.Image) error {
 			if err != nil {
 				return err
 			}
-			absPath, err := config.GetAbsCachePath()
-			if err != nil {
-				return err
-			}
-			cacheDir := filepath.Join(absPath, layout.ImagesDir)
```

Comment:
The main issue was that we were expecting the contents to be under an "images" folder which does not exist in the crane cache